presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -165,6 +165,7 @@
     private Duration fileStatusCacheExpireAfterWrite = new Duration(0, TimeUnit.SECONDS);
     private long fileStatusCacheMaxSize;
     private List<String> fileStatusCacheTables = ImmutableList.of();
+    private String dataCachingDirectory = "";
```

Comment:
why we need a new config despite the one in `CacheConfig`?

Reply:
And it seems unused as well.

Reply:
Removed.